src/modules/fancyzones/lib/JsonHelpers.h

```diff
@@ -59,6 +59,26 @@ namespace JSONHelpers
     using TDeviceInfoMap = std::unordered_map<std::wstring, FancyZonesDataTypes::DeviceInfoData>;
     using TCustomZoneSetsMap = std::unordered_map<std::wstring, FancyZonesDataTypes::CustomZoneSetData>;
 
+    struct MonitorJSON
```

Comment:
MonitorJSON -> MonitorInfo
if `MonitorInfo` is already used for other objects we can come up with a different name.